manage.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+"""
+A django manage.py file.
```

Comment:
I think we don't need it and avoid XBlock as a django project.
It's coming from cookie-cutter though
Thoughts?

Reply:
Does extracting translations work OK without this file?